### PR TITLE
php-fpm support information

### PR DESCRIPTION
**Pull request by enbrnz**
PHP-FPM is not officially supported as the QA teams is running our docker containers which are using mod_php.
SAML SSO with Shibboleth doesn't work using PHP-FPM

Needs backporting

### CODE REVIEW

File: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc [hunk 1]
```diff
@@ -132,6 +132,14 @@ calculate accordingly.
 
 == PHP-FPM
 
+Please note that the ownCloud QA team is using solely mod_php for testing.
```
Comment (voroyam) [inline modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc:135]: Please note that the ownCloud QA team is using solely mod_php for testing.

Note that `mod_php` is used exclusively in the development and QA process of the ownCloud server.

File: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc [hunk 2]
```diff
@@ -132,6 +132,14 @@ calculate accordingly.
 
 == PHP-FPM
 
+Please note that the ownCloud QA team is using solely mod_php for testing. 
+Therefore in a production environment it is recommended to do the same.
```
Comment (voroyam) [inline modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc:136]: Therefore in a production environment it is recommended to do the same.

It's highly recommended to use `mod_php` in your production environment for optimal performance and stability.

File: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc [hunk 3]
```diff
@@ -132,6 +132,14 @@ calculate accordingly.
...
+The ownCloud support team requires the reproduction of any potential cases
```
Comment (voroyam) [inline modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc:137]: The ownCloud support team requires the reproduction of any potential cases
 with mod_php.

Any issues with the ownCloud server have to be reproducible with `mod_php`.

File: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc [hunk 4]
```diff
@@ -132,6 +132,14 @@ calculate accordingly.
...
+with mod_php.
+
+Additionally if you are looking to use SAML SSO with Shibboleth, this
```
Comment (voroyam) [inline modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc:140]: Additionally if you are looking to use SAML SSO with Shibboleth, this
 does not work with PHP-FPM.

SAML SSO with Shibboleth **will not work** with `php-fpm`.

**Review by voroyam — COMMENTED** [review]
Some minor suggestions. In general all the information is there, I just reordered the sentences and added some formatting to it.

Thanks for taking the time and opening a PR @enbrnz 

Let me know if you need help with anything regarding this PR

File: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc [hunk 5]
```diff
@@ -132,6 +132,14 @@ calculate accordingly.
...
+does not work with PHP-FPM.
```
Comment (voroyam) [inline modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc:141]:
```suggestion
SAML SSO with Shibboleth **will not work** with `php-fpm`.
```

**Review by voroyam — COMMENTED** [review]
converted comments in to suggestion that one can commit

File: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc [hunk 6]
```diff
@@ -132,6 +132,14 @@ calculate accordingly.
 
 == PHP-FPM
 
+Please note that the ownCloud QA team is using solely mod_php for testing. 
+Therefore in a production environment it is recommended to do the same.
+The ownCloud support team requires the reproduction of any potential cases
+with mod_php.
```
Comment (voroyam) [inline modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc:138]:
```suggestion
Any issues with the ownCloud server have to be reproducible with `mod_php`
```

**Review by voroyam — APPROVED** [review]

**Comment by voroyam** [comment]
I checked the language and the formatting. Both look good to me.

Erik knows the technical side best.

**Comment by voroyam** [comment]
back ports needed for 10.7 and 10.6

**Comment by phil-davis** [comment]
Backports:
10.7 #3540 
10.6 #3541

**Comment by phil-davis** [comment]
@mmattel this was merged 11 days ago but not yet backported.

What is the process/workflow to make sure that backports are not missed?

**Pull request merged**